.github/workflows/release.yaml

```
@@ -149,4 +149,33 @@ jobs:
           name: v${{ env.VERSION }}
           draft: true
           prerelease: ${{ startsWith(env.VERSION, '0') }}
-          files: artifacts/*
\ No newline at end of file
+          files: artifacts/*
+
+  build-and-push-image:
```

```suggestion
  publish-registry-image:
```

Let's rename this to be more descriptive.